tsl/test/shared/expected/dist_chunk.out

```diff
@@ -0,0 +1,59 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Test function _timescaledb_internal.create_chunk_replica_table
+-- A table for the first chunk will be created on the data node, where it is not present.
+SELECT chunk_name, data_nodes 
+FROM timescaledb_information.chunks 
+WHERE hypertable_name = 'dist_chunk_copy';
+       chunk_name        |        data_nodes         
+-------------------------+---------------------------
+ _dist_hyper_15_67_chunk | {data_node_1,data_node_2}
```

Comment:
These chunk names are not guaranteed to be the same in `shared` test cases.

Reply:
@nikkhils Since the data are inserted in the shared setup, the chunk name is guaranteed.